lesson_13/bank/bank_app/src/main/java/com/codedifferently/lesson13/bank/SavingsAccount.java

```diff
@@ -0,0 +1,15 @@
+package com.codedifferently.lesson13.bank;
+
+import java.util.Set;
+
+public class SavingsAccount extends CheckingAccount {
+
+ // Calls the constructor of the superclass (CheckingAccount) 
+  public SavingsAccount(String accountNumber, Set<Customer> owners, double initialBalance) {
+    super(accountNumber, owners, initialBalance); 
+  }
+
+  public void writeCheck(double amount) {
```

Comment:
If you're going to add this method, you should include a test. More importantly, why even add this if savings account don't support writing checks? Smells like a violation of the "L" in SOLID.